Automattic/woocommerce-payments
# Fix PHPUnit errors for WP 5.9 and PHP 8.x

Fixes #3712

#### Changes proposed in this Pull Request

The purpose of this PR is to fix PHPUnit errors on WP5.9 and PHP 8.x. The reason why errors occurs is that the version of PHPUnit in WooCommerce Payments plugin is incompatible with WP5.9 and PHP 8.x.  To overcome this issue with incompatibility between multiple versions, [PHPUnit Polyfill](https://github.com/Yoast/PHPUnit-Polyfills) will be added as dev dependency. When you see or face an error, you should use TestCase class from this package.


#### Testing instructions

* Make sure that all PHPUnit tests pass on all supported WP versions
* Make sure that all PHPUnit tests pass on PHP 8.x version

-------------------


**Post merge**
